network-api/networkapi/buyersguide/templates/fragments/product_contact.html

```diff
@@ -1,38 +1,40 @@
+{% load i18n %}
+
 <div class="criterion no-border">
   <div class="row mb-3">
     <div class="col-12 col-md-6 mb-3 col-lg-3">
-      <strong class="d-block">Phone Number</strong>
+      <strong class="d-block">{% trans "Phone Number" %}</strong>
       {% if product.phone_number %}
-      {{ product.phone_number }}
+        {{ product.phone_number }}
       {% else %}
-      No
+        {% trans "No" %}
       {% endif %}
     </div>
 
     <div class="col-12 col-md-6 mb-3 col-lg-3 text-truncate">
-      <strong class="d-block">Live Chat</strong>
+      <strong class="d-block">{% trans "Live Chat" %}</strong>
       {% if product.live_chat %}
-      <a id="product-live-chat" target="_blank" href={{ product.live_chat }}>{{ product.live_chat }}</a>
+        <a id="product-live-chat" target="_blank" href="{{ product.live_chat }}">{{ product.live_chat }}</a>
```

Comment:
Caught this one where quotes were missing while fixing my own mistakes

Reply:
I found two more (one is sneaky, it was a data-url attribute instead of href) and added code suggestions for them.